### PR TITLE
Teachers

### PR DESCRIPTION
**Pull request by andrei-81**
added "Our teachers" block

### CODE REVIEW

File: src/css/our-teachers.css [hunk 1]
```diff
@@ -1,3 +1,140 @@
...
+    position: relative;
+  }
+  
+.teachers-underlined-text::after {
+    content: "";
+    position: absolute;
+    left: 0;
+    right: 0;
+    height: 20px; 
+    background-color: #BEE8CC;
```
Comment (Bignichok) [inline src/css/our-teachers.css]: add please this color to css variables and use it from there

File: src/css/our-teachers.css [hunk 2]
```diff
@@ -1,3 +1,140 @@
...
+}
+
+.teachers-text {
+    max-width: 709px;
+    font-family: Manrope;
+    font-size: 14px;
+    font-weight: 400;
+    line-height: 24px;
+    text-align: center;
+    color: #12141780;
```
Comment (Bignichok) [inline src/css/our-teachers.css]: var(--black)
you can see all existed variables in base.css file, apply all colors via varibles

File: src/partials/our-teachers.html [hunk 3]
```diff
@@ -1 +1,60 @@
...
+            Discover our team of dedicated educators, committed to guiding you on your journey to English proficiency with personalized and engaging instruction
+        </p>
+    </div>
+    <div class="teachers-cards-block">
+        <ul class="teachers-list">
+            <li class = teacher-card>
+                <img class="teacher-photo"  
+                    src="./../img/maria.jpeg" 
+                    alt="Maria is thinking about what happened to her dream to be a pet doctor" 
+                    height="150" width="150"/>
```
Comment (Bignichok) [inline src/partials/our-teachers.html]: 1. Create separate folder for teachers images in img folder
2. add please @2x images and use it with picture tag

```
<picture>
                            <source srcset="
                                ./img/teachers/maria.jpg    1x,
                                ./img/teachers/maria@2x.jpg  2x
                            " />
                            <img src="./img/teachers/maria.jpg" alt="Maria is thinking about what happened to her dream to be a pet doctor" />
                        </picture>

File: src/css/our-teachers.css [hunk 4]
```diff
@@ -1,3 +1,140 @@
 div {
     display: block;
 }
```
Comment (Bignichok) [inline src/css/our-teachers.css]: this styles shoud be deleted, it was added becasuse we can not have empty files

File: src/css/our-teachers.css (hunk 1 above)
Comment (andrei-81) [reply src/css/our-teachers.css]: done

File: src/css/our-teachers.css (hunk 2 above)
Comment (andrei-81) [reply src/css/our-teachers.css]: added "dark-gray" and used it

File: src/partials/our-teachers.html (hunk 3 above)
Comment (andrei-81) [reply src/partials/our-teachers.html]: done

File: src/css/our-teachers.css (hunk 4 above)
Comment (andrei-81) [reply src/css/our-teachers.css]: deleted

File: src/css/our-teachers.css [hunk 5]
```diff
@@ -1,3 +1,140 @@
-div {
-    display: block;
+:root {
+    --underline-word: #BEE8CC;
```
Comment (Bignichok) [inline src/css/our-teachers.css]: add it to base css, please

Comment (andrei-81) [reply src/css/our-teachers.css]: done

**Review by Bignichok — APPROVED** [review]

**Pull request merged**